Library/Homebrew/test/cmd/--prefix_spec.rb

```diff
@@ -21,7 +21,14 @@
   end
 
   it "errors if the given Formula doesn't exist", :integration_test do
-    expect { brew "--prefix", "--installed", "nonexistent" }
+    expect { brew "--prefix", "nonexistent" }
+      .to output(/No available formula/).to_stderr
+      .and not_to_output.to_stdout
+      .and be_a_failure
+  end
+
+  it "errors if the given Formula actually matches a Cask", :integration_test do
+    expect { brew "--prefix", "google-chrome" }
```

Comment:
This isn't actually testing the Bash logic (which requires a `brew_sh`)

Reply:
OK, today I learned. :grin:

Reply:
NP, it's a recent change!